packages/integration-tests/create-channel.test.ts

```diff
@@ -101,7 +101,7 @@ describe('Client-Provider-Wallet', () => {
                       '0x00000000000000000000000063E3FB11830c01ac7C9C64091c14Bb6CbAaC9Ac7'
                   }
                 ],
-                assetHolderAddress: '0x4ad3F07BEFDC54511449A1f553E36A653c82eA57'
+                assetHolderAddress: expect.stringMatching(/^0x([a-fA-F0-9]{40})|0$/) // Matches any ethereum address
```

Comment:
The only real change that wasn't deletion. Prior to this change we were using a magic string for this contract address, and it changed under this changeset because the order of contract deployment changed (because I deleted one). Propose that we just match any address here, since the test isn't really concerned with the exact address.